scripts/board_server.py

```diff
@@ -92,7 +92,6 @@ def get_response(request):
 
 
 if __name__ == "__main__":
-
```

Comment:
for some reason this changed is caused by addition of new things to nix, when nix-shell is run